src/components/data-management/Header.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Dropdown, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+
+const { Title, Text, Paragraph } = Typography;
+
+const Header = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata, canLaunchAnalysis,
+    setUploadModalVisible, openAnalysisModal,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const projects = useSelector((state) => state.projects);
+  const changeDescription = (description) => {
+    dispatch(updateProject(activeProjectUuid, { description }));
+  };
+  return (
+    <>
+      <Row style={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Title level={3}>{activeProject.name}</Title>
+        <Space>
+          <Button
+            disabled={projects.ids.length === 0}
+            onClick={() => setUploadModalVisible(true)}
+          >
+            Add samples
+          </Button>
+          <Button
+            disabled={
+              projects.ids.length === 0
+            || activeProject?.samples?.length === 0
+            || isAddingMetadata
+            }
+            onClick={() => {
+              createMetadataColumn();
+            }}
+          >
+            Add metadata
+          </Button>
+          <Dropdown
+            overlay={() => (
+              <DownloadData
+                activeProjectUuid={activeProjectUuid}
+              />
+            )}
+            trigger={['click']}
+            placement='bottomRight'
+            disabled={
+              projects.ids.length === 0
+            || activeProject?.samples?.length === 0
+            }
+          >
+            <Button>
+              Download
+            </Button>
+          </Dropdown>
```

Comment:
Why don't you move all that stuff in `DownloadData` component and in here just call it, passing all the props needed?